addon/data/shooter-interactive-worker.js

```diff
@@ -241,6 +251,10 @@ function restore(state, pos, scrollIntoView) {
       window.scrollTo(0, newScroll);
     }
   }
+
+  if (state === "text") {
+    addHandlers();
```

Comment:
Why addHandlers() for text?  These handlers are for the rectangle selection, they shouldn't be necessary for text.